src/envo/envo-edit.owl

```diff
@@ -39305,6 +39601,2552 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_03501105> "merchant ship"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_03501105> "merchantman"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_03501105> "trading vessel"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501105> "merchant vessel"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501105> <http://purl.obolibrary.org/obo/ENVO_03501325>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501107> (train)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Train>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501107> "A vehicle which locomotes using rails and consists of a series of connected compartments used to transport passengers or cargo."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501107> <https://orcid.org/0000-0002-9578-0788>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501107> "2021-03-15T04:00:01.750Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501107> "train"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501107> <http://purl.obolibrary.org/obo/ENVO_01000604>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501108> (passenger train)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Train#Passenger_trains>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501108> "A train which is capable of and used for passenger transport."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501108> <https://orcid.org/0000-0002-9578-0788>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501108> "2021-03-15T04:00:01.750Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501108> "passenger train"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501108> <http://purl.obolibrary.org/obo/ENVO_03501107>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501109> (subway train)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501109> "A passenger train which runs along an undergroud rail system."@en)
```

Comment:
multiple cases of this

Reply:
@ddooley I think it is probably best to use oboInOwl:source (or dc:source), whatever ENVO uses, instead of hasDBXref..

Reply:
If I'm not mistaken ENVO uses hasdbxref as annotation on definitions to point to definition source.  (Personally, given that a term should have only one definition I have been fine with the IAO:0000118 definition source annotation on the term itself that other projects use).  At any rate I did a pull request https://github.com/EnvironmentOntology/envo/pull/1309  to get rid of the bad hasDBXrefs.

Reply:
>  ENVO uses hasdbxref as annotation on definitions to point to definition source. 

Yes that's our standard, theres always more cleanup but thanks @ddooley for cleaning up in https://github.com/EnvironmentOntology/envo/pull/1309 much obliged!